Maquetado/scss/login.scss

```diff
@@ -0,0 +1,71 @@
+@mixin login-item($margin) {
+  @extend %login-text;
+  margin: $margin;
+  width: 100%;
+}
+
+%login-text {
+  border-radius: 5px;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.login-box {
+  align-items: center;
+  background-color: $wild-sand;
+  display: flex;
+  flex-direction: column;
+  @include border-top($cerulean);
+  justify-content: space-between;
+  margin: 0 auto;
+  max-height: 400px;
+  max-width: 300px;
+  padding: 25px;
+  position: relative;
+  top: 180px;
```

Comment:
Why this value so exact?

Reply:
To make an exact copy from the invision haha